src/libutil/util.hh

```diff
@@ -56,7 +65,7 @@ auto concatStrings(Parts && ... parts)
     -> std::enable_if_t<(... && std::is_convertible_v<Parts, std::string_view>), std::string>
```

Comment:
Rename this one?

Reply:
It was unaffected and I've changed it to use the fixed `concatMapStrings`.

Reply:
Ah I missed that it uses the fixed one

Reply:
It was good to change anyway, no problem/ thanks!